packages/plugins/custom-fields/admin/src/components/ColorPicker/ColorPickerIcon/index.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import icon from './icon.svg';
+
+const ColorPickerIcon = () => <img src={icon} />;
```

Comment:
Could we give the the icon either: 1) an alt text 2) an aria-hidden attribute, if it is decorative only?

Also: should we try and move this icon into the DS? //cc @maevalienard

Reply:
I think actually here we should probably be rendering the svg inline to match the html structure of the other strapi attribute icons.

For the aria-hidden it should probably be on all attribute icons not just custom field icons, right? So maybe I should make that change in the [`AttributeIcon`](https://github.com/strapi/strapi/blob/features/custom-fields/packages/core/content-type-builder/admin/src/components/AttributeIcon/index.js) component, wdyt?

Moving the icon to the DS is fine by me, but I think custom field developers will probably reference this custom-field when building their own so it could be nice to show them that they don't have to use an icon from the DS and how to use an icon that is not in the DS. I guess that example could also just be in the documentation.

Reply:
The objective for now is not to allow external developers to use icons outside of our DS, that's why we added a bunch of random icons in the DS. So I'd rather not go that way, and keep it consistent with the strategy :)

Reply:
So should I add this icon to the DS or use a random one that already exists? @maevalienard could you recommend an icon to use?

Reply:
@markkaylor In case you add a new one, please sync with @HichamELBSI because he will do a release today :)

Reply:
It's already part of the DS under the name "Paint" then :)

Reply:
Awesome! thanks!